orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ArtifactUtils.java

```diff
@@ -227,10 +230,45 @@ public List<Artifact> getArtifactsForPipelineIdWithoutStageRef(
         .orElse(Collections.emptyList());
   }
 
+  private List<String> getExpectedArtifactIdsFromMap(Map<String, Object> trigger) {
+    return Optional.ofNullable((List<String>) trigger.get("expectedArtifactIds"))
+        .orElse(emptyList());
```

Comment:
in simple cases like this, I prefer to use the ternary conditional operator `?:`, we avoid the creation of another object(`Optional`)

Reply:
Okay, I generally avoid ternaries as they can make readability more difficult, but that's a philosophy haha. But yea, let me go ahead and update